source/examples/comparison/main.cpp

```diff
@@ -89,14 +89,14 @@ void compare()
     glbinding_error(false);
 
     std::cout << std::endl << "test: again, now with logging ..." << std::endl;
-    glbinding::logging::log(true);
+    glbinding::logging::start("logs/comparison");
```

Comment:
No, it's the complete path. Therefore, if you run the comparison example multiple times only the last log file will persist and the "logs" directory will not be cluttered with hundreds of log files.